Remove a Linux specific comment from the httping man page.

This change will not be sent upstream.

--- httping-2.4/httping.1.orig	2015-02-16 08:15:41.545818708 -0800
+++ httping-2.4/httping.1	2015-02-16 08:16:09.286228137 -0800
@@ -49,7 +49,7 @@
 Retrieve proxy settings from environment variables ('http_proxy' and 'https_proxy').
 .TP
 .B "\-F"
-Attempt TCP Fast Open while trying to connect to a server (for Linux, version 3.7 onwards of the kernel)   
+Attempt TCP Fast Open while trying to connect to a server.
 .TP
 .B "\-f"
 Flood ping: do not sit idle between each ping but ping as fast as the computer and network allow you to.
